java/kafka/deployment-oauth.yaml

```
@@ -66,35 +68,54 @@ spec:
         app: java-kafka-producer
     spec:
       containers:
-      - name: java-kafka-producer
-        image: quay.io/strimzi-examples/java-kafka-producer:latest
-        env:
-          - name: BOOTSTRAP_SERVERS
-            value: my-cluster-kafka-bootstrap:9092
-          - name: TOPIC
-            value: my-topic
-          - name: DELAY_MS
-            value: "1000"
-          - name: LOG_LEVEL
-            value: "INFO"
-          - name: MESSAGE_COUNT
-            value: "1000000"
-          - name: OAUTH_CLIENT_ID
-            value: java-kafka-producer
-          - name: OAUTH_CLIENT_SECRET
-            valueFrom:
-              secretKeyRef:
-                name: java-kafka-producer-oauth
-                key: clientSecret
-          - name: OAUTH_TOKEN_ENDPOINT_URI
-            value: https://sso.myproject.svc:8443/auth/realms/internal/protocol/openid-connect/token
-          - name: OAUTH_SSL_TRUSTSTORE_TYPE
-            value: PEM
-          - name: OAUTH_SSL_TRUSTSTORE_CERTIFICATES
-            valueFrom:
-              secretKeyRef:
-                name: sso-x509-https-secret
-                key: tls.crt
+        - name: java-kafka-producer
+          image: quay.io/strimzi-examples/java-kafka-producer:latest
+          env:
+            - name: KAFKA_BOOTSTRAP_SERVERS
+              value: my-cluster-kafka-bootstrap:9093
+            - name: STRIMZI_TOPIC
+              value: my-topic
+            - name: STRIMZI_DELAY_MS
+              value: "1000"
+            - name: STRIMZI_LOG_LEVEL
+              value: "INFO"
+            - name: STRIMZI_MESSAGE_COUNT
+              value: "1000000"
+            - name: KAFKA_KEY_SERIALIZER
+              value: "org.apache.kafka.common.serialization.StringSerializer"
+            - name: KAFKA_VALUE_SERIALIZER
+              value: "org.apache.kafka.common.serialization.StringSerializer"
+            - name: OAUTH_CLIENT_ID
+              value: java-kafka-producer
+            - name: OAUTH_CLIENT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: java-kafka-producer
```

To match the name of the secret with the token
```suggestion
                  name: java-kafka-producer-oauth
```